Validation/RecoTrack/python/TrajectorySeedValidation_cff.py

```diff
@@ -0,0 +1,40 @@
+import FWCore.ParameterSet.Config as cms
+
+from Validation.RecoTrack.seedTracks_cfi import seedTracks as _seedTracks
+from SimTracker.TrackAssociation.trackingParticleRecoTrackAsssociation_cfi import trackingParticleRecoTrackAsssociation as _trackAssociation
+from Validation.RecoTrack.TrackValidation_cff import *
+
+_seedProducerLabels = ["initialStepSeeds",
+                       "detachedTripletStepSeeds",
+                       "lowPtTripletStepSeeds",
+                       "pixelPairStepSeeds",
+                       "mixedTripletStepSeedsA",
+                       "mixedTripletStepSeedsB",
+                       "pixelLessStepSeeds",
+                       "tobTecStepSeedsPair",
+                       "tobTecStepSeedsTripl",
+                       "jetCoreRegionalStepSeeds",
+                       ]
+
+_moduleNames = []
+for _label in _seedProducerLabels:
+    _lines = """
+{0}Tracks = _seedTracks.clone(src = cms.InputTag(\"{0}\"))
+{0}Association = _trackAssociation.clone(label_tr = cms.InputTag(\"{0}Tracks\"))
```

Comment:
Since the associations are truly different for all cases, you could avoid creating it altogether here by setting `UseAssociators=True, associators="quickTrackAssociatorByHits"` in MTV. Then MTV will produce the association on the fly (in the regular validation sequence the iteration-specific tracks are subsets of generalTracks, and there it is enough to do the association once).

Then, if you're not too interested of the seed DeltaR distribution or seed fake rate vs. DeltaR, you could use just one MTV instance instead of creating one for each seed collection. (Or if you are interested in these DeltaR plots, I could add a flag which does the DeltaR calculation wrt. tracks of that collection instead of the one specified by `trackCollectionForDrCalculation`).